tests/TranslationRecognizerBasicsTests.ts

```diff
@@ -147,7 +147,8 @@ describe.each([false])("Service based tests", (forceNodeWebSocket: boolean) => {
             ServiceRecognizerBase.telemetryData = undefined;
         });
 
-        test("RecognizeOnceAsync1", (done: jest.DoneCallback) => {
+        // CI test in ADO not receiving recognizing event, disabling for now
```

Comment:
>// CI test in ADO not receiving recognizing event, disabling for now [](http://example.com/codeflow?start=8&length=68)

I believe we changed the native tests to use a longer phrase when we needed a recognizing event.

Reply:
Updated test and re-enabled